src/datetime/DateTime.js

```diff
@@ -121,14 +121,14 @@ export default class Datetime extends React.Component {
 		);
 	}
 
-	renderView( currentView, renderer ) {
+	renderView() {
 		if ( this.props.renderView ) {
-			return this.props.renderView( currentView, () => renderer(currentView) );
+			return this.props.renderView( this.state.currentView, () => this._renderCalendar() );
```

Comment:
We can use `this._renderCalendar` instead of `() => this._renderCalendar()` since `_renderCalendar is already auto bound

Reply:
Done.